sunpy/coordinates/ephemeris.py

```diff
@@ -263,15 +264,26 @@ def _detilt_lon(coord):
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
-def _L0(time='now'):
+def _L0(time='now',
+        light_travel_time_correction=True,
+        nearest_point=True,
+        aberration_correction=False):
     """
-    Return the L0 angle for the Sun at a specified time, which is the Carrington longitude of the
-    Sun-disk center as seen from Earth.
+    Return the L0 angle for the Sun at a specified time, which is the apparent Carrington longitude
```

Comment:
it's only the `apparent` if run with the default values, right?

Reply:
Yes, if you turn off both the light-travel-time correction and the stellar-aberration correction, you will get the true Carrington longitude.

Reply:
OK, then that first sentence could be misleading as to what the function does (or can do).

Reply:
I added a sentence